css-fonts-5/Overview.bs

```diff
@@ -93,13 +93,228 @@ Issue(4044):
 <h3 id="font-size-adjust-prop">
 Relative sizing: the 'font-size-adjust' property</h3>
 
-The 'font-size-adjust' property is applied after the 'size-adjust' descriptor.
+	<pre class="propdef">
+	Name: font-size-adjust
+	Value: none | [ ex | cap | ch | ic ]? [ from-font | <<number>> ]
+	Initial: none
+	Applies to: all elements and text
+	Inherited: yes
+	Percentages: N/A
+	Computed value: the keyword ''font-size-adjust/none'', or a map of the specified keywords and their associated <<number>>
+	Animation type: by computed value type
+	</pre>
+
+	For any given font size, the apparent size and effective legibility of text
+	varies across fonts as a function of their design.
+	For example,
+	for bicameral scripts such as Latin or Cyrillic that
+	distinguish between upper and lowercase letters,
+	the relative height of lowercase letters
+	compared to their uppercase counterparts
+	is a determining factor of legibility.
+	In situations where font fallback occurs,
+	fallback fonts might not share
+	the same ratios as the desired font family
+	for key typographic metrics,
+	and will thus appear to be a different size
+	and possibly be less readable.
+
+	The 'font-size-adjust' property provides a way
+	to preserve the readability and apparent size of text
+	when font fallback occurs.
+	It does this by adjusting the used font size
+	so that the specified metric is the same
+	regardless of the font used.
+
+	Values have the following meanings:
+
+	<dl dfn-type=value dfn-for="font-size-adjust">
+		<dt><dfn>none</dfn>
+		<dd>
+			No special 'font-size' adjustment is applied.
+
+		<dt><dfn>ex | cap | ch | ic</dfn>
```

Comment:
Should this really be marked up as a `<dfn>`? The actual values are defined further down, and that seems enough.

Reply:
It's defining that part of the grammar, isn't it.